util/llpcInternal.cpp

```diff
@@ -77,20 +77,27 @@ Function* GetEntryPoint(
 }
 
 // =====================================================================================================================
-// Emits a LLVM function call (inserted before the specified instruction), builds it automically based on return type
-// and its parameters.
-CallInst* EmitCall(
-    StringRef                     funcName,         // Name string of the function
-    Type*                         pRetTy,           // [in] Return type
-    ArrayRef<Value *>             args,             // [in] Parameters
-    ArrayRef<Attribute::AttrKind> attribs,          // Attributes
-    Instruction*                  pInsertPos)       // [in] Where to insert this call
+/// Emits a LLVM function call using the given builder. The callee is built automically based on return
+/// type and its parameters.
+///
+/// @oaram [in] funcName Name of the callee
+/// @param [in] pRetTy Return type of the callee
+/// @param [in] args Arguments to pass to the callee
+/// @param [in] attribs Function attributes
+/// @param [in] builder The IRBuilder to use for creating the call
+llvm::CallInst* EmitCall(
```

Comment:
Please remove "llvm::" prefix. All source files in LLPC don't use commonly-used namespace unless there is ambiguity. Instead, place "using namespace XXX;" at the beginning of the function.

Reply:
You're right, I'm going to fix this.